code/modules/research/designs/autolathe_designs.dm

```diff
@@ -20,6 +20,15 @@
 	category = list("initial","Tools","Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
 
+/datum/design/broom
+	name="Push Broom"
```

Comment:
Odd name.

Reply:
https://en.wiktionary.org/wiki/push_broom

Reply:
this should to in all_nodes.dm i think?